sdk/servicebus/service-bus/src/serializers/ruleResourceSerializer.ts

```diff
@@ -230,12 +230,23 @@ export class RuleResourceSerializer implements AtomXmlSerializer {
  * @ignore
  * @enum {number}
  */
-enum SqlParameterType {
-  Integer = "l28:int",
-  String = "l28:string",
-  Long = "l28:long",
-  Date = "l28:date"
+enum TypeMapForRequestSerialization {
```

Comment:
So is this a custom serialization scheme that we do for JS or something that all the libraries do (and thus need to coordinate?).

(would be good for a comment here)

Reply:
Service expects the XML request with the special type name, otherwise, the request would fail.
All the languages have to rely on the same.

Will add a comment.

Reply:
Added